projects/storefrontlib/shared/components/card/card.component.html

```diff
@@ -55,7 +55,7 @@ <h3 *ngIf="content.title" class="cx-card-title">
     <div *ngIf="editMode" class="row cx-card-body-delete">
       <div class="col-md-6">
         <button
-          [attr.aria-label]="'screenReader.actionCancelled' | cxTranslate"
+          [attr.aria-label]="'aria.actionCancelled' | cxTranslate"
```

Comment:
This is meant to be an information notification, not a label. Please remove this aria-label